frontend/src/components/Tooltip.js

```diff
@@ -14,6 +14,7 @@ export default function Tooltip({
     setShowTooltip(!showTooltip);
   };
 
+  const svgLineToValue = svgLineTo || 190;
```

Comment:
Could you move the default value to propTypes? (Rather than have it default to null)

Reply:
Updated prop default value.